README.md

```diff
@@ -1 +1,5 @@
-# apple-page
\ No newline at end of file
+# apple-page
+
+This project consists of building a webpage using images as a background and adding gradients to elements. The goal is to make a clone of old an old version of the Apple website webpage.
+
+In this project you’ll clone the website of one of the most design-forward companies in the world. It’s not a very complex site but it uses background photos well and the navbar at the top contains a gradient that you should be able to mimic.
```

Comment:
Let's make the `README` file more descriptive, details like what was done in the projects, what was learned, link to the assignment, clone and source page, screenshots of the clone page can make README look more professional.
 > Remember about descriptive README file.